gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -511,10 +511,12 @@ protected String createAndShowDialog(String current) {
         systemTracingConfigLabel = createLabel(systemTracingConfig, PERFETTO_LABEL);
         Widgets.createButton(systemTracingConfig, "Configure", e -> {
           DeviceCaptureInfo dev = getSelectedDevice();
-          if (dev != null && dev.isFuchsia()) {
+          if (dev == null) {
```

Comment:
This might be a better approach.

```
if (dev != null) {
    if (dev.isFuchsia()) {
            showFuchsiaConfigDialog(getShell(), models, widgets);
     } else {
            showPerfettoConfigDialog(getShell(), models, widgets, dev);
     }
}
```

Reply:
I don't think that's better. It's extra indentation, it doesn't explain why `dev == null` is not being handled, and it doesn't match the same pattern in `updateSystemTracingConfigLabel`.